extra/BUILDING.md

```diff
@@ -40,11 +41,24 @@ To install the native libraries, use the appropriate system package manager.
  * Debian / Ubuntu
     * `sudo apt install libpcre2-dev zlib1g-dev libmbedtls-dev`.
  * Windows (Cygwin)
-    * Run the Cygwin [setup-x86_64.exe](https://cygwin.com/install.html) against the Cygwin installation directory. Install `make`, `git`, `zlib-devel`, `libpcre2-devel`, `mingw64-x86_64-gcc-core`, `mingw64-x86_64-zlib`, and `mingw64-x86_64-pcre2`. You may need to select "Not Installed" in the dropdown list to see the packages. Copy `zlib1.dll` and `libpcre2-8-0.dll` from `path/to/cygwin/usr/x86_64-w64-mingw32/sys-root/mingw/bin` to the checked out Haxe source directory.
+    * Run `opam init` to ensure cygwin is configured with the base packages required. Additional cygwin packages will be installed during the `opam install` further below.
+
+    * Manually install [mingw64-mbedtls](https://github.com/Simn/mingw64-mbedtls) into the cygwin root. First download the correct package from [the releases tab](https://github.com/Simn/mingw64-mbedtls/releases/latest), and then run:
+
+      ```pwsh
+      & $(opam exec -- cygpath -w "/bin/tar") -C / -xvf path/to/package.tar.xz
+      ```
```

Comment:
This fails with:
```
/usr/bin/tar: Child returned status 127
/usr/bin/tar: Error is not recoverable: exiting now
```

Reply:
What is the output if you run these commands:
```
opam exec -- cygpath -w "/bin/tar"
& (opam exec -- cygpath -w "/bin/bash") -c "type tar"
```

Reply:
This was due to conflicts between local cygwin install and opam tar, for some unknown reason